app/javascript/components/training-data/code-tagger/right-pane/RightPane.tsx

```diff
@@ -27,7 +27,7 @@ export function RightPane({
       />
       <button
         onClick={() => confirmTags()}
-        className="btn-m btn-primary mt-auto mb-64"
+        className="btn-m btn-primary mb-32 mt-auto"
```

Comment:
Out of interest, why did you swap these? I tend to prefer margins ordered like they are in CSS shorhand (top/right/bottom/left).

Reply:
I think by pure accident :D I deleted, `mt-auto`, adjusted `mb-32`, then readded `mt-auto`